packages/ruby_debug.rb

```diff
@@ -0,0 +1,63 @@
+# Adapted from Arch Linux ruby-rubocop PKGBUILD at:
+# https://aur.archlinux.org/cgit/aur.git/plain/PKGBUILD?h=ruby-rubocop
+
+require 'package'
+
+class Ruby_debug < Package
+  description 'Ruby debug.rb'
+  homepage 'https://github.com/ruby/debug'
+  version '1.6.2'
+  compatibility 'all'
+  source_url 'SKIP'
+
+  binary_url({
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ruby_debug/1.6.2_armv7l/ruby_debug-1.6.2-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ruby_debug/1.6.2_armv7l/ruby_debug-1.6.2-chromeos-armv7l.tar.zst',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ruby_debug/1.6.2_i686/ruby_debug-1.6.2-chromeos-i686.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ruby_debug/1.6.2_x86_64/ruby_debug-1.6.2-chromeos-x86_64.tar.zst'
+  })
+  binary_sha256({
+    aarch64: '2bdb5f3152576c078f7375bf83cf87c19a620b91c80e6363d127e5aa68ec790c',
+     armv7l: '2bdb5f3152576c078f7375bf83cf87c19a620b91c80e6363d127e5aa68ec790c',
+       i686: '0046afda40659300d9237be3c45ea444f43c1ccadfbc82e787d4528fb55d98c3',
+     x86_64: '449811904d7d9b2969504f5930d3347a4e1590bda94dea9b43d5856935a7a93e'
+  })
+
+  no_fhs
+
+  depends_on 'libyaml'
+  depends_on 'ruby'
+  # depends_on 'xdg_base'
+
+  # @xdg_config_home = ENV.fetch('XDG_CONFIG_HOME', nil)
+  # @xdg_config_home = "#{CREW_PREFIX}/.config" if @xdg_config_home.to_s.empty?
+
+  def self.build
+    # system 'bundle install'
+    # system 'rake build'
+  end
+
+  def self.install
+    # system "gem install --build=#{CREW_DEST_DIR} pkg/rubocop-#{version}.gem"
+    FileUtils.mkdir_p CREW_DEST_PREFIX
+  end
+
+  def self.postinstall
+    @gem_name = name.sub('ruby_', '')
+    system "gem uninstall -Dx --force --abort-on-dependent #{@gem_name} || true"
+    system "gem install -N #{@gem_name} --conservative"
+  end
+
+  def self.remove
+    @gem_name = name.sub('ruby_', '')
```

Comment:
It's fine to use `@gem_name = 'debug'` here. :)  Too much abstraction makes it more difficult to understand.